tfhe/src/high_level_api/integers/types/base.rs

```diff
@@ -166,6 +168,47 @@ where
     }
 }
 
+impl<P, T> FheTryTrivialEncrypt<T> for GenericInteger<P>
+where
+    T: Into<U256>,
+    P: IntegerParameter<
+        InnerCiphertext = RadixCiphertextDyn,
+        InnerServerKey = crate::integer::ServerKey,
+    >,
+    P::Id: WithGlobalKey<Key = GenericIntegerServerKey<P>> + Default,
+{
+    type Error = crate::high_level_api::errors::Error;
+
+    fn try_encrypt_trivial(value: T) -> Result<Self, Self::Error> {
+        let value = value.into();
+        let id = P::Id::default();
+        let ciphertext = id.with_global(|key| match key.pbs_order {
+            crate::shortint::PBSOrder::KeyswitchBootstrap => {
+                RadixCiphertextDyn::Big(key.inner.create_trivial_radix(value, key.num_block))
+            }
+            crate::shortint::PBSOrder::BootstrapKeyswitch => {
+                RadixCiphertextDyn::Small(key.inner.create_trivial_radix(value, key.num_block))
+            }
```

Comment:
can we maybe add a small test somewhere to check we are getting the right CT type for trivial encryption ? Or maybe just an assert in the u128 and u256 tests you wrote to check we are getting big/small as expected?

Reply:
I'll add one

Reply:
added asserts